dspace/src/main/docker-compose/README.md

```diff
@@ -181,6 +181,7 @@ The remainder of these instructions assume you are using ngrok (though other pro
 Prerequisites
 - Start DSpace 7 using one of the options listed above
 - Build the DSpace CLI image if needed.  See the instructions above.
+- A collection of AIPs to load into your instance of DSpace. You can either [make these yourself from another instance of DSpace](https://github.com/DSpace-Labs/AIP-Files#to-create-test-data), or you can use a pre-built set. We have [a few you can use](https://github.com/DSpace-Labs/AIP-Files#sample-datasets), but probably the most useful is the [set of AIPs we use for our DSpace demo site](https://github.com/DSpace-Labs/AIP-Files/releases/tag/demo-site-aips).
```

Comment:
@hardyoyo : Just to clarify... if you follow the instructions in this section, you are not able to select a different set of AIPs.  There is a hardcoded set of AIPs in the `cli.ingest.yml` script (which is in the instructions below)...it's hardcoded to use the AIPs at `https://github.com/DSpace-Labs/AIP-Files/raw/main/dogAndReport.zip`

See https://github.com/DSpace/DSpace/blob/main/dspace/src/main/docker-compose/cli.ingest.yml#L14

So, I'm not sure if this is customizable at this time.  I fully agree it'd be *nice* to make this customizable, but I haven't tested whether it's possible to use a different set of AIPs.

Reply:
Closing this, since the actual problem seems to be that the unzip command is no longer present in the CLI container. I may write up docs on how to customize the cli.ingest.yml file to load whatever AIPs you happen to have at hand, but that's a larger project for another day.

Reply:
fix for the actual problem I was having:
https://github.com/DSpace/DSpace/pull/8807